downstream/modules/automation-hub/proc-delete-namespace.adoc

```diff
@@ -4,18 +4,18 @@
 
 = Deleting a namespace
 
-You can delete unwanted namespaces to manage storage on your {hubname} server. To do so, ensure that the namespace does not contain a collection with dependencies.
+You can delete unwanted namespaces to manage storage on your {HubName} server. To do so, ensure that the namespace does not contain a collection with dependencies.
 
 .Prerequisites
-* The namespace being deleted does not have a collection with dependencies.
+* The namespace you are deleting does not have a collection with dependencies.
 * You have *Delete namespace* permissions.
 
 .Procedure
-. Log in to your local {hubname}.
+. Log in to your local {HubName}.
 . Navigate to menu:Collections[Namespaces].
 . Click the namespace to be deleted.
 . Click image:more_actions.png[More actions], then click btn:[Delete namespace].
 +
-NOTE: If the btn:[Delete namespace] button is disabled, it means that this namespace contains a collection with dependencies. Review the collections in this namespace and delete any dependencies to proceed with the namespace deletion. See _Managing collections on automation hub_ in the link:https://access.redhat.com/documentation/en-us/red_hat_ansible_automation_platform[Red Hat Ansible Automation Platform documentation] for information about deleting collections.
+NOTE: If the btn:[Delete namespace] button is disabled, it means that this namespace contains a collection with dependencies. Review the collections in this namespace and delete any dependencies to proceed with the namespace deletion. See link:https://access.redhat.com/documentation/en-us/red_hat_ansible_automation_platform/2.3/html-single/managing_red_hat_certified_and_ansible_galaxy_collections_in_automation_hub/index[Managing Red Hat Certified and Ansible Galaxy collections in automation hub] for information about deleting collections.
 
-The deleted namespace, as well as its associated collections, will now be deleted and removed from the namespace list view.
+The namespace that you deleted, as well as its associated collections, is now deleted and removed from the namespace list view.
```

Comment:
Although if it's a namespace you're deleting, you want Section 1.7 of Curating collections using namespaces in Automation Hub.